integration-cli/docker_cli_run_test.go

```diff
@@ -4284,6 +4284,15 @@ func (s *DockerSuite) TestRunCredentialSpecWellFormed(c *check.C) {
 func (s *DockerSuite) TestRunServicingContainer(c *check.C) {
 	testRequires(c, DaemonIsWindows, SameHostDaemon)
 
+	// This functionality does not exist in post-RS3 builds. Note we get the version number
+	// this way as test binaries are not manifested.
```

Comment:
Nit: Instead of saying "this way" you could be more precise with something like "from the full build string", and perhaps add that these binaries will get Windows 8 version information: 6.2, build 9200.

Reply:
Updated (and added MSDN link)

Reply:
Thanks, looks good. Too bad the recommended version helper functions do not help with the build number. Keeping the same major and minor version numbers forever seems like a bad idea.